src/orderbook.js

```diff
@@ -1197,6 +1336,44 @@ OrderBook.prototype.computeAutobridgedOffers = function() {
   this._offersAutobridged = autobridgeCalculator.calculate();
 };
 
+OrderBook.prototype.computeAutobridgedOffersWrapper = function() {
+  var startTime = Date.now();
+  this.computeAutobridgedOffers();
+  this.mergeDirectAndAutobridgedBooks();
+  var lasted = (Date.now() - startTime);
+  
+  const newMult =
+    ((lasted * 2 / OrderBook.AUTOBRIDGE_CALCULATE_THROTTLE_TIME) << 0) + 1;
```

Comment:
> Maybe you have other processes running on your system? It is consistent for me.

I don't know what's wrong with jsperf, but simple tests that just do `Math.floor` in loop shows that's it significantly slower than shifting or `| 0` in firefox, but slightly faster in node.js

> I don't follow this. The only difference is whether it gets truncated to 32 bits right?

I just mentioned this because I think it can be better optimized by engine